accel_code/run.py

```diff
@@ -67,6 +67,8 @@ def main():
     # add the handlers to the logger
     logger.addHandler(fh)
     logger.addHandler(ch)
+    # add new log header
+    logger.info('##########')
```

Comment:
```suggestion
    logger.info('\n##########\n')
```
would that add newlines surrounding the hashes? that would make the distinction between logs even more clear.

Reply:
yup that makes sense!